centaur/src/it/scala/centaur/AbstractCentaurTestCaseSpec.scala

```diff
@@ -189,8 +189,12 @@ abstract class AbstractCentaurTestCaseSpec(cromwellBackends: List[String], cromw
 
     runTestIo.redeemWith(
       {
-        case centaurTestException: CentaurTestException => maybeRetry(centaurTestException)
-        case _ => runTestIo
+        case centaurTestException: CentaurTestException =>
+          maybeRetry(centaurTestException)
+        case nonCentaurException =>
```

Comment:
Despite the name and from a quick scan, I'm not sure if this variable's type is actually `java.lang.Exception`. I don't know the full monadic variable origin of this value, and if at runtime it truly will be an Exception and not something else.

Are there any cases of `java.lang.Error` to try and process here? What do you think about explicitly typing this to only filter `java.lang.Exception` cases? The same "only handle `java.lang.Exception`" would go for a number of the other changes in this PR as well.

Reply:
It's type is `Throwable`. I don't think we need some special handling for `Error` here - if it happens we are doomed anyway and adding a special handling would just make the code a bit more verbose

Reply:
But I renamed the variable and added type in the case clause in order to improve readability